scripts/README.md

```diff
@@ -0,0 +1,22 @@
+# Scripts
+
+Helper scripts for SWIM maintenance tasks.
+
+## Installation
+
+- Requires Python 3
+- Install dependencies with `pip install -r requirements.txt`
+- Configure custom CA in `~/.aws/config` (if required)
+```toml
+[default]
+ca_bundle = "<...>.pem"
+```
+
+## Execute
+
+```sh
+# help
+python ./set_tags.py --help
+# example
+python ./set_tags.py --endpoint-url http://localhost:9000 --bucket swim-bucket --prefix test-meta/inProcess/ --tagging '{"SWIM_State": "processed"}'
```

Comment:
Maybe also add a few words about the parameter `tagging`. That one cannot process a full JSON structure (as this would not translate into S3 tags), but in fact only a flat key-value-structure.